migrations/20190509095330_add_wif_to_numberMapping.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex, Promise) {

```

Comment:
This looks good now! Just missing the rollback.

```
exports.down = function(knex, Promise) {
  return Promise.all([
    knex.schema.table('numberMapping', (table) => {
      table.dropColumn('WIF');
    }),
  ]);
}
```

Reply:
Rollback added.

Reply:
Rollback added.